models/users.js

```diff
@@ -372,7 +376,13 @@ const addForVerification = async (userId, discordId, profileImageUrl, discordIma
     userId,
     discordId,
     discord: { url: discordImageUrl, approved: false, date: admin.firestore.Timestamp.fromDate(new Date()) },
-    profile: { url: profileImageUrl, approved: false, date: admin.firestore.Timestamp.fromDate(new Date()) },
+    profile: {
+      url: profileImageUrl,
+      publicId: profileImageID,
+      approved: false,
+      date: admin.firestore.Timestamp.fromDate(new Date()),
```

Comment:
What date formate?

Reply:
date: {
        _seconds: 1686518413,
        _nanoseconds: 453000000,
      },
      
      this the format, as it was already in user, I didn't changed as it may lead for me to handle the case for the previous objects.